plugins/modules/vmware_cluster_drs_recommendations.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: vmware_cluster_drs_recommendations
+short_description: Apply DRS Recommendations
+description:
+    - Apply DRS Recommendations for Cluster.
+author:
+- Nina Loser (@Nina2244)
+options:
+    cluster_name:
+      description:
+      - The name of the cluster to be managed.
+      type: str
+      required: true
+    datacenter:
+      description:
+      - The name of the datacenter.
+      type: str
+      required: true
+      aliases: [ datacenter_name ]
+extends_documentation_fragment:
+- community.vmware.vmware.documentation
+
+'''
+
+EXAMPLES = r'''
+- name: Apply DRS Recommendations for Cluster
+  community.vmware.vmware_cluster:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    datacenter_name: datacenter
+    cluster_name: cluster
+  delegate_to: localhost
+'''
+
+RETURN = r'''
+result:
+    description:
+        - list of the recommendations
+        - What server moved from which host to which host.
+    returned: always
+    type: list
+    sample: ["server1 move from host1 to host2.", "server2 move from host1 to host2."]
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.community.vmware.plugins.module_utils.vmware import (
+    PyVmomi,
+    find_datacenter_by_name,
+    find_cluster_by_name,
+    vmware_argument_spec,
```

Comment:
```suggestion
    find_datacenter_by_name,
    vmware_argument_spec,
```

The CI complains that `find_cluster_by_name` isn't used.